README.md

```diff
@@ -86,3 +94,67 @@ Syntax for the `import` task:
 This task does not accept a `-c` parameter, because it implicitly relates to running the import scripts.
 
 If the `-n` parameter is specified, it is the equivalent of running the `nuke` task followed by the `import` task.
+
+## Options Summary
+
+The use of standard and additional parameters is summarised in the following table:
+
+| Task     | `-e <environment>` | `-c <component>` | Additional parameters (mostly optional) |
+|----------|--------------------|------------------|-----------------------------------------|
+| `build`  |                    | Required         | `-t <target-commitish>`; `-r <release>` |
+| `deploy` | Optional`*`        | Required         | `-r <release>` (Required)               |
+| `nuke`   | Optional`*`        |                  |                                         |
+| `import` | Optional`*`        |                  | `-n`                                    |
+
+`*` If the `-e <environment>` parameter is missing, the task will operate locally.
+
+## GitHub Access Token Generation
+
+The `build` task requires a GitHub access token to be generated and set as the value of the `RWAHS_ACCESS_TOKEN`
+environment variable.  To create a token, use the following command syntax:
+
+    curl -u <username> --data '{"note":"<note>","scopes":["repo"],client_id":"<client_id>","client_secret":"<client_secret>","fingerprint":"<fingerprint>"}' https://api.github.com/authorizations
```

Comment:
That's weird because I'm pretty sure I copied and pasted that from the command I ran, which worked. Perhaps the github api is tolerant of malformed json? 

Will fix.